src/global.js

```diff
@@ -24,6 +24,94 @@ const color = {
   YELLOW: '\x1b[33m'
 };
 
+/**
+ * Convenience wrapper for preset console messaging and colors.
+ *
+ * @type {{warn: (function(...[*]): void), log: (function(...[*]): void), success: (function(...[*]): void),
+ *    error: (function(...[*]): void), info: (function(...[*]): void)}}
+ */
+const consoleMessage = (() => {
+  const applyColor = (method, passedColor, ...args) =>
+    console[method](`${passedColor}${args.join('\n')}${color.NOCOLOR}`);
+
+  return {
+    error: (...args) => applyColor('error', color.RED, ...args),
+    success: (...args) => applyColor('log', color.GREEN, ...args),
+    info: (...args) => applyColor('info', color.BLUE, ...args),
+    log: (...args) => applyColor('log', color.NOCOLOR, ...args),
+    warn: (...args) => applyColor('warn', color.YELLOW, ...args)
+  };
+})();
+
+/**
+ * Basic message logging.
+ *
+ * @type {{readonly guides: Array, log: Function, _log: object, readonly messages: Array, error: Function,
+ *     message: Function, readonly logs: Array, readonly errors: Array, guide: Function}}
+ */
+const logger = {
```

Comment:
required:
lets break out logger into a separate standalone commit and then implement the `guide` option